crates/matrix-sdk/src/guaranteed_completion.rs

```diff
@@ -0,0 +1,262 @@
+// Copyright 2022 The Matrix.org Foundation C.I.C.
```

Comment:
You're supposed to put your own name here :)

Unless you don't want to own the copyright, in which case you probably have to state that somewhere more explicitly (but I am not a lawyer and don't know for sure).

Reply:
should I add my name to all the files I've amended or only the new one?

Reply:
Ig it's just updating call sites of some API you changed, no. If you added or rewrote significant amounts of code, then yes. That said, I forget to do it all the time myself, it's a best-effort thing.